test/controllers/FeedbackFormsControllerSpec.scala

```diff
@@ -332,13 +332,68 @@ class FeedbackFormsControllerSpec extends PlaySpecification with TestEnvironment
 
     "build json from case class" in new WithTestApplication {
       val questions = Question("how is knolx portal?", List("awesome", "i can do it better"))
-
       val feedbackForm = FeedbackForm("test", List(questions), active = true, BSONObjectID.parse("5943cdd60900000900409b26").get)
-
       val result = controller.jsonCountBuilder(feedbackForm)
       result must be equalTo """{"0":"2"}"""
     }
 
+    "generate feedback form preview for success" in new WithTestApplication {
```

Comment:
A better test description would be "get feedback form" or something similar as used in other test descriptions.